__tests__/app.test.js

```diff
@@ -262,4 +262,43 @@ it('400: should return an error when the user enters an empty object', () => {
 })
 })
 
+describe("DELETE /api/comments/:comment_id", () => {
+    it('204: should allow the user to delete a comment by the given comment_id and should return no content', () => {
+        request(app)
+        .delete('/api/comments/2')
+        .expect(204)
+        .then((result) => {
+            expect(result.body).toEqual({})
+        })
+
+        return request(app)
+        .delete('/api/comments/2')
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe("Id does not exist")
+        })
```

Comment:
You might find sometimes this fails - ideally you'd need to do this new async request, inside the .then on 270 - that way you know its been deleted, before doing the request to check if its gone. 

.then((result) => {
            expect(result.body).toEqual({})

return request(app)
        .delete('/api/comments/2')
        .expect(404)
        }) 
.then(({ body }) => {
            expect(body.msg).toBe("Id does not exist")
        })